client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx

```diff
@@ -49,6 +49,14 @@ const MasteringGutenberg = () => {
 					showIcon={ false }
 					text={ translate( 'Adjusting settings of blocks' ) }
 				/>
+				<InlineSupportLink
+					supportPostId={ 145498 }
+					supportLink={ localizeUrl( 'https://support.wordpress.com/free-photo-library' ) }
+					showIcon={ false }
+					text={ translate(
+						'Add free beautiful copyright-free photos to create stunning designs'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'The WordPress.com Free Photo Library integrates your site with beautiful copyright-free photos to create stunning designs.' )` *ES Score: 7*